### ISSUE TITLE
Publish to Hex.pm

### ISSUE THREAD

**Issue by guilleiguaran** [yakaz/yamerl#13]
The repository has already a package.exs file, there is a reason that has not been published in http://hex.pm yet?

**Comment by amorphid** [comment]
I would also like to see this published to Hex.  I even forked it, just so I could publish the fork on Hex.

If you like, I can publish yamerl on Hex myself.  I didn't want to do so without your blessing!

**Comment by amorphid** [comment]
it'd be nice if projects like [Yamlelixir](https://github.com/KamilLelonek/yaml-elixir#installation) could install directly from Hex, instead of cloning this repo on GitHub :)

**Comment by dumbbell** [comment]
Hi! Sorry, I have little time to work on this project now that I'm no longer employed by Yakaz.

So I tried to publish the package this morning but I get a 500 HTTP error. Investigating...

**Comment by dumbbell** [comment]
Ok, it's fixed now:
* I had to remove non-US-ASCII characters in `src/yamerl.app.src` (several "é" in my name).
* I had rebar3_hex 1.19.0 installed. `rebar3 plugins upgrade rebar3_hex` said it upgraded the plugin to 2.5.1 but 1.19.0 was still the only one in `~/.cache/rebar3/plugins`. After removing that copy and running rebar3 again, it fetched the expected 2.5.1 version.

The package is now published to Hex.pm.

Thank you for reminding me to finally do this! It was easier than I imagined.
